edit/edit.go

```diff
@@ -682,6 +682,18 @@ func DictionarySet(dict *build.DictExpr, key string, value build.Expr) build.Exp
 	return nil
 }
 
+// DictionaryGet looks for the key in the dictionary expression, and returns the
+// current value. If it is unset, it returns nil.
+func DictionaryGet(dict *build.DictExpr, key string) build.Expr {
+	for _, e := range dict.List {
+		kv, _ := e.(*build.KeyValueExpr)
```

Comment:
while it should be a KeyValueExpr, might as well inspect and continue if not

kv, ok := e.(*build.KeyValueExpr)
if !ok {
  continue
}